feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SpeakerFragment.kt

```diff
@@ -53,6 +55,8 @@ class SpeakerFragment : DaggerFragment() {
         ) { speaker ->
             binding.speaker = speaker
         }
+        binding.speakerDescription.autoLinkMask = Linkify.WEB_URLS
```

Comment:
TextView#setAutoLinkMask does not run Linkify automatically so it would not work if the text had already been set.
It would be nice if you call `LinkifyCompat.addLinks`.

Reply:
ok, I understood. I'll change from `autoLinkMask` to `LinkifyCompat.addLinks`

Reply:
@jmatsu  fixed. Please review this PR🙇